src/main/java/seedu/address/model/person/Person.java

```diff
@@ -89,30 +95,22 @@ public boolean equals(Object other) {
         }
 
         Person otherPerson = (Person) other;
-        return otherPerson.getName().equals(getName())
-                && otherPerson.getPhone().equals(getPhone())
-                && otherPerson.getEmail().equals(getEmail())
-                && otherPerson.getAddress().equals(getAddress())
-                && otherPerson.getTags().equals(getTags());
+        return otherPerson.getNric().equals(getNric()) && otherPerson.getName().equals(getName())
+                && otherPerson.getPhone().equals(getPhone()) && otherPerson.getEmail().equals(getEmail())
+                && otherPerson.getAddress().equals(getAddress()) && otherPerson.getTags().equals(getTags());
     }
 
     @Override
     public int hashCode() {
         // use this method for custom fields hashing instead of implementing your own
-        return Objects.hash(name, phone, email, address, tags);
+        return Objects.hash(nric, name, phone, email, address, tags);
     }
 
     @Override
     public String toString() {
         final StringBuilder builder = new StringBuilder();
-        builder.append(getName())
-                .append(" Phone: ")
-                .append(getPhone())
-                .append(" Email: ")
-                .append(getEmail())
-                .append(" Address: ")
-                .append(getAddress())
-                .append(" Tags: ");
+        builder.append(getNric()).append(" Name: ").append(getName()).append(" Phone: ").append(getPhone())
+        .append(" Email: ").append(getEmail()).append(" Address: ").append(getAddress()).append(" Tags: ");
```

Comment:
/nit Personally I feel the old style is clearer.

Reply:
Oh it's just the default Eclipse formatter.

Do you want me to revert this?

Reply:
Yep I think put a chaining method to a different line looks cleaner.